src/coretelephony.cs

```diff
@@ -80,8 +82,29 @@ interface CTTelephonyNetworkInfo {
 		CTCarrierEventHandler CellularProviderUpdatedEventHandler { get; set; }
```

Comment:
Good thing it's deprecated because we should probably have kept the `Subscriber` prefix.